app/cowsay.html

```diff
@@ -0,0 +1,14 @@
+<div>
+
+  {{ $ctrl.title  }}
+
+  <input type = "text" ng-model= "$ctrl.userInput">
+
+    <pre>
+
+      {{ $ctrl.cowsayBrowser($ctrl.userInput)  }}
```

Comment:
Your ``cheese`` is not appearing properly when I view your page on ``locahost:8080``. You have to change Line 9 to something else. Look closely at your ``entry.js`` and try to figure out what needs to go on Line 9 instead.